widgets/widgetindex.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: Flutter Widget Index
+sidebar: home_sidebar
+permalink: widgets/widgetindex/
+---
```

Comment:
Can you add the following text:

"This is an alphabetical list of nearly every widget that is bundled with Flutter. You can also <a href="/widgets/">browse widgets by category</a>."